docker-compose.yml

```diff
@@ -36,17 +36,17 @@ services:
       - NODE_ENV=development
     volumes:
       - ./workspace-zen/cp-zen-platform:/usr/src/app:Z
-      - ./workspace-zen/cp-zen-frontend:/usr/src/app/node_modules/cp-zen-frontend:Z
-      - ./workspace-zen/cp-translations:/usr/src/app/node_modules/cp-translations:Z
+      - ./workspace-zen/cp-zen-frontend:/usr/src/app/node_modules/cp-zen-frontend:ro
```

Comment:
The Z is for fedora selinux having stricter policies.

Reply:
Only needed if we want to write to the folder, which is why I've gone `ro` in the first place. We've had an issue with our `.git` folder disappearing from cp-zen-frontend and cp-translations. Unable to reliably reproduce, but this seems likely to be the culprit given it doesn't happen to the services.

Reply:
Ah yes and write is needed as you've a seperate file for yarn now

Reply:
Good assumption